extra/modules/confiant-ad-quality/src/test/java/org/prebid/server/hooks/modules/com/confiant/adquality/v1/ConfiantAdQualityBidResponsesScanHookTest.java

```diff
@@ -218,6 +218,60 @@ public void shouldSubmitToScanBidsWhichAreNotPartOfTheExcludeToScanConfig() {
         )));
     }
 
+    @Test
+    public void shouldSubmitToScanOnlyBidsWithData() {
```

Comment:
the names of the tests doesn't fit our code style rules, please rename them to the following pattern:

`(methodName)Should(expectedResult)When(condition or case)`

Reply:
It means I need to rewrite all test names inside this module. Do you want me to do in the current PR?

Reply:
This is an open-source community, and it's better to follow the common code style. I'm not going to block the merging only because of it, but eventually, especially if you continue working on the module, it's worth coming to the same code style. 

We usually use the Boy Scout Rule, and if you're able to improve at least something, please do it. The scale of improvement doesn't matter. It could the one line, one class or the whole module. As you wish)

Reply:
Got it. But I would like to push tests renaming as a separate PR if you don't mind.
So I'm gonna fix your other notes and create a separate PR with renaming of all test cases inside the Module.
Just to track it easier. Thanks.